### PR TITLE
Updated editor setup scripts

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
- Removed some unused methods and cleaned up the class a bit.
- Enables shared depth buffer for Windows Mixed Reality in the project by default.

Changes
---
- Fixes: #3158
- Fixes: #1939
- [x] Fixes: #3259

Blocked by
- [ ] #3269

### CODE REVIEW

**Comment by StephenHodgson** [comment]
@wassx I didn't intend to try and fix #3183 with this change, but can you test it for me?

File: Assets/MixedRealityToolkit/_Core/Utilities/Editor/Setup/.MixedRealityEditorSettings.cs [hunk 1]
```diff
@@ -169,49 +176,18 @@ private static bool FindDirectory(string directoryPathToSearch, string directory
...
-            }
-
-            AssetDatabase.SaveAssets();
-            AssetDatabase.Refresh(ImportAssetOptions.ForceUpdate);
-        }
+        #region IActiveBuildTargetChanged Implementation
 
         /// <inheritdoc />
-        public int callbackOrder { get; private set; }
+        public int callbackOrder => 0;
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Utilities/Editor/Setup/.MixedRealityEditorSettings.cs]:
```suggestion
        public int callbackOrder { get { return 0; } }
```

**Comment by alcooper91** [comment]
Hi Stephen, I won’t be able to test this. 
@davidkline-ms could you or Kurtis test this or have someone follow up?

**Comment by wassx** [comment]
I'll test today. Thx!

**Comment by wassx** [comment]
Sry, the issue is still there: _'sharedassets0.assets' is corrupted!_

**Comment by wassx** [comment]
Yep, deleted Library and my build directories, so unity reloaded everything

**Comment by StephenHodgson** [comment]
> StabilizationPlaneModifier 

We don't have that in vNEXT

**Comment by jbienzms** [comment]
Hey @keveleigh do you know anyone who can tell us exactly what the right thing is to do when Shared Depth Buffer is enabled for RS4? Should we disable  `StabilizationPlaneModifier` or set it's `SetStabilizationPlane` field to false? Or maybe something else?

**Comment by jbienzms** [comment]
> > StabilizationPlaneModifier
> 
> We don't have that in vNEXT

How does the stabilization plane get set in vNEXT? Is RS4 or later required for vNEXT?

**Comment by StephenHodgson** [comment]
> RS4 or later required for vNEXT

That is correct. We designed it with the assumption that the stabilization plane will not be modified by the app.

**Comment by jbienzms** [comment]
Okay. I can approve the current fix assuming that only RS4+ will be used.

**Comment by StephenHodgson** [comment]
Although I think you're right Jarrod, we should call out that we're setting the stabilization plane in the dialog.

Let me make that change before you approve.

**Comment by StephenHodgson** [comment]
Done :)

**Pull request closed without merging**